staff/jordi-pujol/array-functions/splice.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Abstraction of splice
+ * 
+ * changes the contents of an array by removing or replacing existing elements and/or adding new elements.
+ * 
+ * @param {Array} arr 
+ * @param {Number} start 
+ * @param {Number} countDel 
+ * @param {*} item
+ * 
+ * @throws {}
+ * 
+ * @returns {Array} An array containing the deleted elements.
+ */
+
+function splice(arr, start, countDel, item) {
```

Comment:
splice good implemented ✌, i think you can refactor it without difficulties

Reply:
Yes! Thanks for the feedback! I've just changed it now :)